modules/cloudtrail__download_event_history/main.py

```diff
@@ -3,30 +3,23 @@
 import json
 import time
 
+from core.lib import strip_lines, save, downloads_dir
+from pacu import Main
 
 module_info = {
-    # Name of the module (should be the same as the filename)
     'name': 'cloudtrail__download_event_history',
-
-    # Name and any other notes about the author
     'author': 'Spencer Gietzen of Rhino Security Labs',
-
-    # Category of the module. Make sure the name matches an existing category.
     'category': 'EVADE',
-
-    # One liner description of the module functionality. This shows up when a user searches for modules.
     'one_liner': 'Downloads CloudTrail event history to JSON files.',
-
-    # Description about what the module does and how it works
-    'description': 'This module will download the CloudTrail event history for each specified region in both JSON format to ./sessions/[current_session_name]/downloads/cloudtrail_[region]_event_history_[timestamp].json. Warning: This module can take a very long time to complete because the maximum events per API call is 50, when there could be tens or hundreds of thousands or more total events to download. A rough estimate is about 10000 events retrieved per five minutes.',
-
-    # A list of AWS services that the module utilizes during its execution
+    'description': strip_lines('''
+        This module will download the CloudTrail event history for each specified region in both JSON format to 
+        ~/.local/share/pacu/sessions/[current_session_name]/downloads/cloudtrail_[region]_event_history_[timestamp].json.
+        Warning: This module can take a very long time to complete because the maximum events per API call is 50, when 
+        there could be tens or hundreds of thousands or more total events to download. A rough estimate is about 10000
+        events retrieved per five minutes.
+    '''),
```

Comment:
This doesn't show up in GitHub too well but before this was just a single long line. I suppose soft wrapping was probably enabled before, but with python's explicit white space it's best to make this explicit.